src/sanity_test.rs

```diff
@@ -0,0 +1,28 @@
+use std::cmp::PartialOrd;
+
+macro_rules! test_template {
```

Comment:
Hey, thanks for your awesome contribution. Would you consider to use https://crates.io/crates/rstest instead of writing a macro here?

Reply:
Resolved.